clients/client-shell/cmds/version/version.go

```diff
@@ -2,12 +2,40 @@
 package version
 
 import (
-	"fmt"
-
+    "fmt"
+    "io/ioutil"
+    "net/http"
+    "encoding/json"
+    "runtime"		
 	"github.com/spf13/cobra"
 	"github.com/taskcluster/taskcluster/clients/client-shell/cmds/root"
 )
 
+
+// Useful structure for assets[] in the returned json data 
+type Assets struct {
+    Download string                  `json:"browser_download_url"`
+   }
+  
+   // Useful structure in the returned json data 
+  
+  type Release struct {
+      Name string                    `json:"name"`
+      Asslist []Assets                `json:"assets"`
```

Comment:
It's usually best to make the struct name mirror the JSON, name, with proper case, so here `Assets`.